main_routers/live2d_router.py

```
@@ -0,0 +1,907 @@
+    """上传Live2D模型到用户文档目录"""
+    _config_manager = get_config_manager()
+    
+    try:
+        if not files:
+            return JSONResponse(status_code=400, content={"success": False, "error": "没有上传文件"})
+        
+        # 创建临时目录来处理上传的文件
+        with tempfile.TemporaryDirectory() as temp_dir:
+            temp_path = pathlib.Path(temp_dir)
+            
+            # 保存所有上传的文件到临时目录，保持目录结构
```

Ruff (0.14.8)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)